File: src/main/java/net/volcano/jdacommands/model/command/Field.java
package net.volcano.jdacommands.model.command;

import net.dv8tion.jda.api.entities.MessageEmbed;

public class Field {
	
	public final String name;
	
	public final String value;
	
	public final Boolean inline;
	
	public Field(String name, String value, Boolean inline) {
		this.name = name;
		this.value = value;
		this.inline = inline;
	}
	
	public Field(String name, String value) {
		this.name = name;
		this.value = value;
		inline = true;
	}
	
	public MessageEmbed.Field build() {
		return new MessageEmbed.Field(name, value, inline);
	}
	
}

File: src/main/java/net/volcano/jdacommands/model/command/Field.kt
package net.volcano.jdacommands.model.command

import net.dv8tion.jda.api.entities.MessageEmbed

class Field(
	val name: String,
	val value: String,
	val inline: Boolean = false
) {

	fun build(): MessageEmbed.Field {
		return MessageEmbed.Field(name, value, inline)
	}
}